velox/expression/SimpleFunctionAdapter.h

```diff
@@ -511,6 +511,12 @@ class SimpleFunctionAdapter : public VectorFunction {
     }
   }
 
+#if __GNUC__ == 8
+#define FIX_GCC8_BUG , this
```

Comment:
This is a bit ugly. @laithsakka Is there a better way? Do we have to support gcc 8.x?

Reply:
I know 😕

GCC 8 is needed to support Debian 10, which is what Dataproc (Google's EMR) uses.
Using a later GCC version is nontrivial, as there is a versioning issue with `libstc++`.

Another way we can do it is check for the compiler version in the Makefile, and remove `-Wunused-lambda-capture` when it's supported by the compiler (Clang)

Reply:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100295